### ISSUE TITLE
Huge output in continuous integration environments

### ISSUE THREAD

**Issue by malyw** [dgarlitt/karma-nyan-reporter#16]
Jenkins output example: ![Jenkins output](http://i.imgur.com/It1Z2Iz.png)
It would be nice to have an optional config param to suppress drawing until the end of the test run.

**Comment by malyw** [comment]
As possible workaround for a while- environment variables can be used in `karma.conf.js`:
```js
{
 ...
 reporters: process.env.DEV ? ['nyan'] : ['progress'],
 ...
}
```

**Comment by ldeveber** [comment]
I'm also seeing this issue. :(
<img width="820" alt="screen shot 2016-03-09 at 12 39 37 pm" src="https://cloud.githubusercontent.com/assets/4457752/13644511/22368416-e5f4-11e5-837f-7080958807d2.png">

**Comment by dgarlitt** [comment]
This issue has been addressed in the latest release [v0.2.4 on npm](https://www.npmjs.com/package/karma-nyan-reporter). Thanks for the feedback! @malyw Sorry it took so long for me to get around to adding it, thanks for you patience. Changes pertaining to this issue can be found here: dae17fe3ec39a395f374f86c5b0dd72f97e36843
